src/accelerate/utils/imports.py

```diff
@@ -210,7 +210,19 @@ def is_tqdm_available():
 
 
 def is_mlflow_available():
-    return _is_package_available("mlflow")
+    package_exists = importlib.util.find_spec("mlflow") is not None
+    if package_exists:
+        try:
+            _ = importlib.metadata.metadata("mlflow")
+            return True
+        except importlib.metadata.PackageNotFoundError:
+            # support a case of mlflow installed via mlflow-skinny
+            try:
+                _ = importlib.metadata.metadata("mlflow-skinny")
+                return True
+            except importlib.metadata.PackageNotFoundError:
+                return False
+    return False
```

Comment:
```suggestion
    return _is_package_available("mlflow") or _is_package_available("mlflow-skinny")
```

Reply:
@BenjaminBossan @muellerzr Thank you for your comments.

I found the below behaviour.

![Uploading image.png…]()

I did not come with simple code.

Reply:
I see. Would this work:

```python
def is_mlflow_available():
    if _is_package_available("mlflow"):
        return True

    if importlib.util.find_spec("mlflow") is not None:
        try:
            _ = importlib.metadata.metadata("mlflow-skinny")
            return True
        except importlib.metadata.PackageNotFoundError:
            return False
    return False
```